rmc/resources/grch37/rmc.py

```diff
@@ -0,0 +1,374 @@
+"""
+Script containing RMC and MPC related resources.
+
+RMC: Regional missense constraint
+MPC: Missense badness, Polyphen-2, and Constraint score
+"""
+import hail as hl
+
+from gnomad.resources.resource_utils import TableResource, VersionedTableResource
+
+from rmc.resources.basics import (
+    CONSTRAINT_PREFIX,
+    MODEL_PREFIX,
+    MPC_PREFIX,
+    RESOURCE_PREFIX,
+)
+from rmc.resources.resource_utils import CURRENT_VERSION
+
+
+FREEZES = [1, 2]
+"""
+List of RMC/MPC data versions.
+"""
+
+CURRENT_FREEZE = 2
+"""
+Current RMC/MPC data version.
+"""
+
+
+# Original regional missense constraint resource files
+MUTATION_RATE_TABLE_PATH = f"{RESOURCE_PREFIX}/GRCh37/exac/mutation_rate_table.tsv"
```

Comment:
Same thing I mentioned in another comment — maybe we can save `f"{RESOURCE_PREFIX}/GRCh37/exac"` in a variable to reuse in these paths